EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -15237,7 +15237,7 @@ QUEST_LV_0100_20160718_015236	Ah well, I suppose it can't be helped.{nl}Could I
 QUEST_LV_0100_20160718_015237	There are some precious corals growing near the Kranta Skerries.{nl}Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods.{nl}

```

Comment:
I believe the blue coral might have connections to the `Greenish Stone` that was potentially named `Cerulean Stone` and  also the `Crafted Coral`. I'll make a note about it for the future and give it your suggested change.